engine/topology/src/main/java/org/opennms/oce/engine/topology/ActionManager.java

```diff
@@ -63,7 +63,8 @@ public void createReport(Group group) {
     }
 
     public void createSituation(ReportObject report) {
-        ImmutableSituation.Builder situationBuilder = ImmutableSituation.newBuilder().setId(report.getId());
+        ImmutableSituation.Builder situationBuilder = ImmutableSituation.newBuilderNow()
```

Comment:
This is what is breaking the test, the test relied on both situations having the same creation time before.

```suggestion
        ImmutableSituation.Builder situationBuilder = ImmutableSituation.newBuilder()
                .setCreationTime(0).
```